repose-aggregator/src/docs/asciidoc/architecture/performance.adoc

```diff
@@ -48,6 +48,59 @@ As Gatling runs, it will pipe the performance data it captures to InfluxDB.
+The data being transferred will include numerous statistics calculated for a distinct time.
+To get a better idea of the data in question, a sample has been provided which captures the number of request/response pairs sent at time `1492629621` as well as various statistics calculated from the amount of time taken to service those requests.
+Note that the sample data only captures a single second of the Gatling simulation.
+Much more data of the same form would be expected for every second that the Gatling simulation ran.
+
+....
+gatling.computerworld.getComputers.ok.count 2 1492629621
+gatling.computerworld.getComputers.ok.min 192 1492629621
+gatling.computerworld.getComputers.ok.max 219 1492629621
+gatling.computerworld.getComputers.ok.mean 205 1492629621
+gatling.computerworld.getComputers.ok.stdDev 13 1492629621
+gatling.computerworld.getComputers.ok.percentiles50 192 1492629621
+gatling.computerworld.getComputers.ok.percentiles75 219 1492629621
+gatling.computerworld.getComputers.ok.percentiles95 219 1492629621
+gatling.computerworld.getComputers.ok.percentiles99 219 1492629621
+gatling.computerworld.getComputers.ko.count 0 1492629621
+gatling.computerworld.getComputers.all.count 2 1492629621
+gatling.computerworld.getComputers.all.min 192 1492629621
+gatling.computerworld.getComputers.all.max 219 1492629621
+gatling.computerworld.getComputers.all.mean 205 1492629621
+gatling.computerworld.getComputers.all.stdDev 13 1492629621
+gatling.computerworld.getComputers.all.percentiles50 192 1492629621
+gatling.computerworld.getComputers.all.percentiles75 219 1492629621
+gatling.computerworld.getComputers.all.percentiles95 219 1492629621
+gatling.computerworld.getComputers.all.percentiles99 219 1492629621
+....
+
+==== InfluxDB
+
+InfluxDB will process the data it receives through its Graphite listener port by applying configured templates.
+These templates will translate the metric names from the Graphite line protocol into measurements, fields, tags, and values in the database.
+
+For example, given the sample data provided in the <<Gatling, Gatling section>>, the following template could be applied:
+
+....
+gatling.*.*.*.* measurement.simulation.request.status.field
+....
+
+Resulting in a database full of entries like the following:
+
+....
+name: gatling # <1>
+time                count max mean min percentiles50 percentiles75 percentiles95 percentiles99 request                  simulation    status stdDev
+----                ----- --- ---- --- ------------- ------------- ------------- ------------- -------                  ----------    ------ ------
+1492629621000000000 2     219  205 192 192           219           219           219           getComputers           computerworld all    12
+....
+<1> The name of the measurement.
+
+To learn more about how InfluxDB handles Graphite input, see the https://github.com/influxdata/influxdb/blob/master/services/graphite/README.md[Graphite Input README].
```

Comment:
Good point on also potentially going with "see [Graphite Input]." That being said, I can also see what you're saying, too. I'm fine with leaving it as is.